recipes/gametes/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "gametes" %}
+{% set version = "2.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+about:
+  home: https://sourceforge.net/projects/gametes/
+  license: GNU General Public License version 2.0 (GPLv2)
+  summary: Tool for the generation of complex single SNP models
+  license_file: LICENSE
+
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+  - {{ pin_subpackage("gametes", max_pin="x") }}
+
+source:
+  url: https://sourceforge.net/projects/gametes/files/GAMETES_2.1.jar
+  sha256: 5e38b0993f8915e3e3e4c225ce6b684455d02c6189877d6ae219d24ad69e27d7
+
+requirements:
+  run:
+    - openjdk >=17
+    - python
+  host:
+    - openjdk >=17
+    - python
+
+test:
+  commands:
+    - $(echo $(java -jar $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/gametes.jar -h |  grep Usage))
```

Comment:
A test for the wrapper script would also be a good idea, as well as mentioning the wrapper script in a note (e.g. see the [peptide-shaker recipe](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/peptide-shaker/meta.yaml)) so that users will know how to run the program.